src/app/pages/about/about.component.scss

```
@@ -4,303 +4,146 @@ $padding: 11%;
 
-          h4 {
-            font-size: 23px;
-            margin: 15px 0;
-          }
+        .event {
+          border-radius: 20px;
+          background-color: white;
 
-          img {
-            height: 100px;
-            width: auto;
+          mat-card-actions {
+            button{
+              background-color: white;
+            }
           }
         }
-      }
-    }
-
-    &.goals-container, &.alfy-contacts {
-      background-color: $accent-darker;
-      color: white;
-      z-index: 2;
-
-      &:before, &:after {
-        display: block;
-        position: relative;
-        content: "";
-        width: 100% * (1 / 0.78);
-        left: 11% * (-1 / 0.78);
-        height: 24px;
-        background: url(/assets/img/shadow.png) no-repeat top center;
-        background-size: contain;
-      }
-
-      &:before {
-        top: 0;
-      }
-
-      &:after {
-        bottom: 0;
-        transform: rotate(180deg);
-      }
-
-      .title {
-        margin-top: 10px;
-      }
-    }
-
-    &.alfy-actions {
-      $gutter: 30px;
-      $border-width: 5px;
-      $dot-diameter: 8px;
 
-      display: flex;
-      justify-content: center;
+        .example-header-image {
+          background-image: url("/assets/img/logo-alfy.jpg");
+          background-size: cover;
+        }
 
-      * {
-        box-sizing: border-box;
-      }
+        .title{
+          text-align: center;
+        }
 
-      .timeline {
-        width: 100%;
-        max-width: 800px;
-        padding: 20px 0;
-        position: relative;
-        margin: 0 auto 30px;
+        .box {
+          text-align: center;
+          color: white;
+          padding: 10px;
+          width: 70%;
+          margin: auto;
+        }
 
-        &:before {
-          content: '';
-          position: absolute;
-          top: -15px;
-          left: calc( 34% - 3px );
-          height: calc( 100% + 45px );
-          bottom: 0;
-          width: 4px;
-          background: $accent-darker;
-          z-index: 1;
+        .test {
+          font-size: 100;
         }
       }
 
-      .entry {
-        text-align: left;
-        display: flex;
-        align-items: baseline;
-
-        .title {
-          float: left;
-          flex: 34%;
-          padding-right: $gutter;
-          text-align: right;
-          position: relative;
-          margin: 0 !important;
-
-          &:before {
-            content: '';
-            position: absolute;
-            width: $dot-diameter;
-            height: $dot-diameter;
-            border: $border-width solid $warn;
-            background-color: #fff;
-            border-radius:100%;
-            top: 15%;
-            right: -$dot-diameter - 0.5px;
-            z-index: 99;
-          }
-
-          h3 {
-            margin: 0;
-            font-size: 120%;
-
-            @media only screen and (max-width: 500px) {
-              font-size: 100% !important;
+      .low-right-mock-up{
+        .presentation{

```

Pour fixer la hauteur de l'agenda à celle du reste de la page, partie 2: 
```suggestion
        .presentation{
        flex: 0 0 64%;
        @media only screen and (min-width: 768px) {
          margin-left: 36%;
        }
```